src/components/CampsiteListItem.js

```diff
@@ -39,6 +39,8 @@ const CampsiteListItem = ({site, getSiteDetail, navigate}) => {
     const {textStyle, IconContainer, subtitleView, siteAvatarStyle, siteAvatarContainerStyle, titleView, nearestTownStyle, badgeWrapperStyle, badgeContainerStyle, accessibilityStyle} = styles
     const {campsite_form: {accessibility_options}} = campsite;
     const {id, title, description, nearestTown, accessibility, siteImageData, features, facilities} = site;
+    const preparedDescription = description && description.length > 57 ?  `${description.substring(0, 57)}...` : description;
```

Comment:
Ellipsized descriptions so that the ListItem view description is never more than 57 characters long :) This nicely lines up with the same length of description we get for the Markers on the MapView when we click on a site.